mockito/matchers.py

```diff
@@ -246,18 +246,22 @@ def __repr__(self):
 class ArgumentCaptor(Matcher):
     def __init__(self, matcher=None):
         self.matcher = matcher or Any()
-        self.value = None
+        self.values = list()
```

Comment:
Made this `values`. I am ok to change it to `all_values` if this attribute is very similar to `value` property.

Reply:
To have `values` with and without the `s` is unfortunate; but actually `value` is the misnomer in the first place, as it should be "most-recent-value".  Esp. when you do `verify(thing, times=3).action(capt)`, t.i. we expect the method to be called multiple times, it is clear that "value" is a sloppy name.

Reply:
I agree. But I guess, `value` is close to Java mockito ArgumentCaptor:`getValue`(If it's still the goal). I could rename the `values` to `all_values` to match with the naming convention used here https://javadoc.io/static/org.mockito/mockito-core/4.1.0/org/mockito/ArgumentCaptor.html#getAllValues--